diploshic/fvTools.py

```diff
@@ -916,7 +921,7 @@ def calcAndAppendStatValDiplo(
             - statVals["pi"][instanceIndex][subWinIndex]
         )
     elif statName == "HapCount":
-        statVals[statName][instanceIndex].append(len(hapsInSubWin.distinct()))
+        statVals[statName][instanceIndex].append(len(genosInSubWin.distinct()))
```

Comment:
hey @bruce-edelman -- slightly worried about this change? does this not throw off the Haplotype stuff?

Reply:
Ah, I may have been confused but looking at the code the Haplo/diplo functions seem to be split and each called when specifying haplo or diplo modes. In the [haplo](https://github.com/kr-colab/diploSHIC/blob/master/diploshic/fvTools.py#L704) version there is the variable `hapsInSubWin` provided but in the [diplo](https://github.com/kr-colab/diploSHIC/blob/master/diploshic/fvTools.py#L864) version of the same function there isn't. So that means that this if statement must never be true otherwise there would be undefined variable errors. 

I had (maybe overconfidently) assumed that this function maybe was copied from the haplo version and this variable was meant to be `genosInSubWin`